homeassistant/components/deconz/sensor.py

```diff
@@ -166,11 +207,18 @@ def async_add_sensor(
             ):
                 entities.append(DeconzSensor(sensor, gateway))
 
-            if sensor.secondary_temperature:
-                known_temperature_sensors = set(gateway.entities[DOMAIN])
-                new_temperature_sensor = DeconzTemperature(sensor, gateway)
-                if new_temperature_sensor.unique_id not in known_temperature_sensors:
-                    entities.append(new_temperature_sensor)
+            for sensor_description in SENSOR_DESCRIPTIONS:
+
+                try:
+                    if sensor_description.value_fn(sensor):
+                        known_sensors = set(gateway.entities[DOMAIN])
+                        new_sensor = DeconzPropertySensor(
+                            sensor, gateway, sensor_description
+                        )
+                        if new_sensor.unique_id not in known_sensors:
+                            entities.append(new_sensor)
+                except AttributeError:
```

Comment:
When can the AttributeError happen?

Reply:
@MartinHjelmare i changed this afterwards. There is no longer any try except. It was if the attribute wouldn't exist